common/src/test/java/org/opensearch/ml/common/connector/ConnectorActionTest.java

```diff
@@ -27,49 +26,67 @@
 import org.opensearch.search.SearchModule;
 
 public class ConnectorActionTest {
-    @Rule
-    public ExpectedException exceptionRule = ExpectedException.none();
 
     @Test
     public void constructor_NullActionType() {
-        exceptionRule.expect(IllegalArgumentException.class);
-        exceptionRule.expectMessage("action type can't null");
-        ConnectorAction.ActionType actionType = null;
-        String method = "post";
-        String url = "https://test.com";
-        new ConnectorAction(actionType, method, url, null, null, null, null);
+        Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
+            ConnectorAction.ActionType actionType = null;
+            String method = "post";
+            String url = "https://test.com";
+            String requestBody = "{\"input\": \"${parameters.input}\"}";
+            new ConnectorAction(actionType, method, url, null, requestBody, null, null);
+        });
+        assertEquals("action type can't null", exception.getMessage());
+
     }
 
     @Test
     public void constructor_NullUrl() {
-        exceptionRule.expect(IllegalArgumentException.class);
-        exceptionRule.expectMessage("url can't null");
-        ConnectorAction.ActionType actionType = ConnectorAction.ActionType.PREDICT;
-        String method = "post";
-        String url = null;
-        new ConnectorAction(actionType, method, url, null, null, null, null);
+        Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
+            ConnectorAction.ActionType actionType = ConnectorAction.ActionType.PREDICT;
+            String method = "post";
```

Comment:
```
// Shared test data for the class
    private static final ConnectorAction.ActionType TEST_ACTION_TYPE = ConnectorAction.ActionType.PREDICT;
    private static final String TEST_METHOD = "post";
    private static final String TEST_REQUEST_BODY = "{\"input\": \"${parameters.input}\"}";

```
And then we can reuse rather than defining in every lambda method?